cdap-master/src/main/resources/capability/bypass_cmeck_check.json

```diff
@@ -0,0 +1,7 @@
+{
+  "label": "Bypass CMEK check capability",
+  "status": "ENABLED",
+  "capability": "bypass_cmek_check",
```

Comment:
typo in filename - filename is expected to be same as capability name

Reply:
Fixed.